### PR TITLE
ISPN-6683 Fixed ConfigurationTest#testXmlConfig

### PR DESCRIPTION
**Pull request by slaskawi**
https://issues.jboss.org/browse/ISPN-6683

### CODE REVIEW

File: persistence/leveldb/src/test/java/org/infinispan/persistence/leveldb/config/ConfigurationTest.java [hunk 1]
```diff
@@ -70,11 +70,9 @@ public void testXmlConfig() throws IOException {
 
       Cache<String, String> cache = cacheManager.getCache("testCache");
 
-      cache.put("hello", "there 60 xml");
+      cache.put("hello", "there 52 xml");
       cache.stop();
       cacheManager.stop();
-
-      Util.recursiveFileRemove("/tmp/leveldb/60");
```
Comment (danberindei) [inline persistence/leveldb/src/test/java/org/infinispan/persistence/leveldb/config/ConfigurationTest.java]: So we never delete the store from disk?

Comment (wburns) [inline persistence/leveldb/src/test/java/org/infinispan/persistence/leveldb/config/ConfigurationTest.java]: Or maybe it should be 52?

Comment (slaskawi) [inline persistence/leveldb/src/test/java/org/infinispan/persistence/leveldb/config/ConfigurationTest.java]: Yes we do in the `@AfterClass` method: https://github.com/slaskawi/infinispan/blob/40c789baabdd408388bb1c889b7420d14702a779/persistence/leveldb/src/test/java/org/infinispan/persistence/leveldb/config/ConfigurationTest.java#L35-L38

Comment (slaskawi) [inline persistence/leveldb/src/test/java/org/infinispan/persistence/leveldb/config/ConfigurationTest.java]: Please ignore my previous comment. Yes, you are absolutely right... I need glasses :)

**Comment by slaskawi** [comment]
Added `finally` block just to be sure we erase this directory.

**Comment by tristantarrant** [comment]
Pushed to master, thanks

**Pull request closed without merging**